### PR TITLE
10 get users

### PR DESCRIPTION
**Pull request by RGBlife**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -295,3 +295,74 @@ describe("POST /api/articles/:article_id/comments", () => {
...
+    const {users} = response.body
+
+    const expected = {
+      avatar_url: "https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg",
+      name: "jonny",
+      username: "butter_bridge",
+    };
+
+    expect(users[0]).toMatchObject(expected);
+    expect(users).toHaveLength(4);
```
Comment (two-triangles) [inline __tests__/app.test.js:502]: We are only checking one user here. We should be following the pattern of checking the shape of all of the user objects that come back and that the typeof of the property values is what we can expect. 

We can use a forEach for this 👍

File: models/users.model.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection");
+
+exports.fetchUsers = async () => {
+  try {
+    const usersQuery = `SELECT * FROM users`;
+    const { rows } = await db.query(usersQuery);
+    return rows
+  } catch (error) {
```
Comment (two-triangles) [inline models/users.model.js]: Same as on previous regarding have two catch blocks

Comment (RGBlife) [reply models/users.model.js]: Removed

File: __tests__/app.test.js (hunk 1 above)
Comment (RGBlife) [reply __tests__/app.test.js:502]: Added the forEach method to check each property value, thank you! :)

**Review by Beth-McCormack — APPROVED** [review]

**Pull request merged**